fs/sshfs/sshfs.py

```diff
@@ -155,6 +156,28 @@ def listdir(self, path):  # noqa: D102
         with convert_sshfs_errors('listdir', path):
             return self._sftp.listdir(_path)
 
+    def scandir(self, path, namespaces=None, page=None):  # noqa: D102
+        self.check()
+        _path = self.validatepath(path)
+        _namespaces = namespaces or ()
+        _start, _stop = page or (None, None)
```

Comment:
I know it's a bit of a bad habit from my side, but you don't have to prefix variables with underscore everywhere; I try to do it for things that are already in scope (`_path` being the absolute validated version of `path`), but I sometimes do it even for other things. The correct Python practice AFAIK is to use an underscore *suffix* instead but I think it looks ugly :smile: